sim/common/sod/item_effects/phase_6.go

```diff
@@ -0,0 +1,122 @@
+package item_effects
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/stats"
+)
+
+const (
+	SignetRingDominatorR5   = 234034
+	SignetRingDominatorR4   = 234030
+	SignetRingDominatorR3   = 234026
+	SignetRingDominatorR2   = 234021
+	SignetRingDominatorR1   = 234017
+	SignetRingInvokerR5     = 234032
+	SignetRingInvokerR4     = 234028
+	SignetRingInvokerR3     = 234024
+	SignetRingInvokerR2     = 234020
+	SignetRingInvokerR1     = 234016
+	SignetRingFlamekeeperR5 = 234964
+	SignetRingFlamekeeperR4 = 234965
+	SignetRingFlamekeeperR3 = 234966
+	SignetRingFlamekeeperR2 = 234967
+	SignetRingFlamekeeperR1 = 234968
+	SignetRingPreserverR5   = 234033
+	SignetRingPreserverR4   = 234029
+	SignetRingPreserverR3   = 234025
+	SignetRingPreserverR2   = 234023
+	SignetRingPreserverR1   = 234019
+	SignetRingProtectorR5   = 234035
+	SignetRingProtectorR4   = 234031
+	SignetRingProtectorR3   = 234027
+	SignetRingProtectorR2   = 234022
+	SignetRingProtectorR1   = 234018
+	SignetRingSubjugatorR5  = 234436
+	SignetRingSubjugatorR4  = 234437
+	SignetRingSubjugatorR3  = 234438
+	SignetRingSubjugatorR2  = 234439
+	SignetRingSubjugatorR1  = 234440
+	SignetRingConquerorR5   = 234202
+	SignetRingConquerorR4   = 234201
+	SignetRingConquerorR3   = 234200
+	SignetRingConquerorR2   = 234199
+	SignetRingConquerorR1   = 234198
```

Comment:
Normally I like to keep these ordered by ID but I think this makes a lot of sense like this here 👍 keeps it much more readable